cluster/addons/fluentd-elasticsearch/es-image/go.mod

```diff
@@ -33,3 +33,5 @@ require (
 	k8s.io/utils v0.0.0-20181221173059-8a16e7dd8fb6 // indirect
 	sigs.k8s.io/yaml v1.1.0 // indirect
 )
+
+go 1.13
```

Comment:
Not sure how this file got added in (maybe somewhere in the build process?) but I don't think it should be part of this PR.

Reply:
Thanks! my ide is dumb and keeps adding this